docker-compose.yml

```diff
@@ -7,17 +7,31 @@ services:
       - 9432:9432
     restart: always
 
+  gitcollector:
+    image: srcd/gitcollector:v0.0.1-beta.2
```

Comment:
`beta.3` has been released.

Reply:
we need to keep beta.2 for now. beta.3 requires a new configuration. it will be addressed later.